Cargo.toml

```diff
@@ -15,6 +15,11 @@ name = "rusoto"
 readme = "README.md"
 repository = "https://github.com/rusoto/rusoto"
 version = "0.24.0"
+homepage = "https://www.rusoto.org/"
+
+[badges]
+travis-ci = { repository = "rusoto/rusoto", branch = "master" }
+appveyor = { repository = "rusoto/rusoto" }
```

Comment:
I wasn't clear on this earlier: can we get the Appveyor badge to also specify `master`?  After that we're good to go! 👍